### ISSUE TITLE
Enhancement - clear custom coordinate formats, reset UI to default

### ISSUE THREAD

**Issue by BobBooth** [Esri/coordinate-conversion-addin-dotnet#449]
It might be useful to be able to clear out custom coordinate formats, to reset the UI of the add-in to default (uninstall/reinstall does not seem to do this).
Would need to put the UI for this somewhere a bit out of the way to keep people from accidentally wiping out their coordinate formats.

**Comment by topowright** [comment]
This currently does not make sense to do now. Assigning this to icebox for that reason.

**Comment by topowright** [comment]
@lfunkhouser I do not think this is something that needs to be implemented and should be closed as no fix.

**Comment by lfunkhouser** [comment]
@dfoll going to review, update.
